common/wonders/wc_wonders_historical.txt

```diff
@@ -621,3 +620,105 @@ wonder_ulduar = {
 		}
 	}
 }
+
+#Janeiro Point
+wonder_janeiro_point = {
```

Comment:
The upgrades look a bit weird to me.
Like if it's a statue, why not add the statue upgrades?